src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1375,41 +1376,65 @@ uct_dc_mlx5_iface_dci_do_rand_pending_tx(ucs_arbiter_t *arbiter,
     return res;
 }
 
-static ucs_arbiter_cb_result_t
-uct_dc_mlx5_ep_arbiter_purge_cb(ucs_arbiter_t *arbiter, ucs_arbiter_group_t *group,
-                                ucs_arbiter_elem_t *elem, void *arg)
+ucs_arbiter_cb_result_t
+uct_dc_mlx5_ep_arbiter_purge_internal_cb(ucs_arbiter_t *arbiter,
+                                         ucs_arbiter_group_t *group,
+                                         ucs_arbiter_elem_t *elem, void *arg)
 {
     uct_purge_cb_args_t *cb_args = arg;
     void **priv_args             = cb_args->arg;
     uct_dc_mlx5_ep_t *ep         = priv_args[0];
```

Comment:
you can assume `arg` is a pointer to ep, you do not need to pass an array of args here

Reply:
fixed